docs/get_started/index.md

```diff
@@ -11,141 +11,70 @@ imperative operations on the fly.  A graph optimization layer on top of that
 makes symbolic execution fast and memory efficient. The MXNet library is
```

Comment:
1. overview is not severing as testing installation. it gives an quick overview of mxnet's interface. we can provide some test scripts for people to test. or simply make test
2. @zackchase is working on it
3. noop. @zackchase suggested to move the installation into the mxnet.io frontpage to reduce one mouse click
4. placing on the left is also ugly. i don't have a good solution yet. but i probably left it to our future web designers.